relnotes

```diff
@@ -294,6 +302,8 @@ generate_notes () {
   local tempfile=/tmp/$PROG-file-1.$$
   local anchor
   local changelog_file
+  local search_pattern="Merge pull"
+  local search_field="4"
```

Comment:
what is this?

Reply:
Refactored and commented that a bit to make it more clear.